code/controllers/subsystem/statistics.dm

```diff
@@ -6,7 +6,7 @@ SUBSYSTEM_DEF(statistics)
 
 
 /datum/controller/subsystem/statistics/Initialize(start_timeofday)
-	if(!config.sql_enabled)
+	if(!GLOB.configuration.database.enabled)
```

Comment:
TODO: Change to `SSdbcore.IsConnected()`

Reply:
may want to add that and an actual todo comment so you don't forget  later.

Reply:
Oh it was meant for doing in this PR, I am just too dumb to read my own review